bgpd/bgp_fsm.c

```diff
@@ -218,7 +218,6 @@ static struct peer *peer_xfer_conn(struct peer *from_peer)
 			 * due to connection transfer.
 			 */
 			stream_free(peer->curr);
-			peer->curr = NULL;
```

Comment:
why are we removing the NULL set?  Else we have the peer->curr pointing to freed memory

Reply:
It should be freed by stream_free() -> XFREE(), right?